CONTRIBUTING.md

```diff
@@ -3,13 +3,13 @@
 ## Raising defects
 
 Before raising an issue, make sure you have checked the open and closed issues to see if an answer is provided there.
-There may also be an answer to your question on [stackoverflow](stackoverflow.com/questions/tagged/pact).
+There may also be an answer to your question on [stackoverflow](https://stackoverflow.com/questions/tagged/pact).
 
 Please provide the following information with your issue to enable us to respond as quickly as possible.
 
 1. The relevant versions of the packages you are using.
 1. The steps to recreate your issue.
-1. An executable code example where possible. You can fork this repository and modify the e2e [example](https://github.com/pact-foundation/pact-go/blob/master/dsl/pact_integration_test.go)  to quickly recreate your issue.
+1. An executable code example where possible. You can fork this repository and modify the e2e [example](https://github.com/pact-foundation/pact-go/blob/master/dsl/pact_test.go)  to quickly recreate your issue.
```

Comment:
Not sure if this is correct - `pact_test.go` was the closest I could find to the integration test.

Reply:
Good spot! Just point them at the `examples` folder - they are all E2E tests of sorts. Running `make pact` will execute these.

Reply:
ok sounds good